.ci-mgmt.yaml

```diff
@@ -8,3 +8,14 @@ plugins:
   - name: terraform
     version: "1.0.16"
     kind: converter
+
+worktreeAllowedChanges: |-
+  sdk/**/pulumi-plugin.json
+  sdk/dotnet/Pulumi.*.csproj
```

Comment:
Why are you ignoring these? I didn't see a similar thing needed in the pulumi-random PR?

Shouldn't these resolve to the same thing as in the PR?